### PR TITLE
Lifter solenoid DONT CR

### PR DESCRIPTION
**Pull request by greenblitz4590**

### CODE REVIEW

File: src/main/java/edu/greenblitz/robotName/subsystems/Lifter/LifterSolenoid/SimulationLifterSolenoid/SimulationLifterSolenoid.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+	@Override
+	public void holdSolenoid() {
+		setVoltage(LifterSolenoidConstants.POWER_TO_HOLD * RobotConstants.SimulationConstants.BATTERY_VOLTAGE);
+	}
+
+	private void setVoltage(double voltage){
+		appliedOutputs = voltage;
+		simulationMotor.setInputVoltage(voltage);
+	}
```
Comment (bossyshrimp68) [inline src/main/java/edu/greenblitz/robotName/subsystems/Lifter/LifterSolenoid/SimulationLifterSolenoid/SimulationLifterSolenoid.java:40]: space

File: src/main/java/edu/greenblitz/robotName/subsystems/Lifter/LifterSolenoid/SimulationLifterSolenoid/SimulationLifterSolenoidConstants.java [hunk 2]
```diff
@@ -0,0 +1,6 @@
+package edu.greenblitz.robotName.subsystems.Lifter.LifterSolenoid.SimulationLifterSolenoid;
+
+public class SimulationLifterSolenoidConstants {
+
+	public static final int MOTOR_NUM = 1;
```
Comment (bossyshrimp68) [inline src/main/java/edu/greenblitz/robotName/subsystems/Lifter/LifterSolenoid/SimulationLifterSolenoid/SimulationLifterSolenoidConstants.java:5]:
```suggestion
	public static final int NUMBER_OF_MOTORS = 1;
```

**Review by bossyshrimp68 — CHANGES_REQUESTED** [review]

File: src/main/java/edu/greenblitz/robotName/OI.java [hunk 3]
```diff
@@ -105,11 +94,11 @@ public void thirdJoystickButtons(){
     }
 
     public void initializeDefaultCommands() {
-        SwerveChassis.getInstance().setDefaultCommand(new MoveByJoysticks(DRIVE_MODE));
+//        SwerveChassis.getInstance().setDefaultCommand(new MoveByJoysticks(DRIVE_MODE));
```
Comment (yonnbro) [inline src/main/java/edu/greenblitz/robotName/OI.java:97]: Don’t comment out controls in master!!

File: src/main/java/edu/greenblitz/robotName/commands/Climb/lifter/ExtendLifter.java [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+    public void initialize() {
+        lifterSolenoid.closeSolenoid();
+        timer.start();
+    }
+
+    @Override
+    public void execute() {
+        if (timer.hasElapsed(LifterSolenoidConstants.SECONDS_TO_CLOSE)){
+            lifterSolenoid.holdSolenoid();
+//            lifter.goToPosition(LifterConstants.LIFTER_EXTENDED_POSITION);
```
Comment (yonnbro) [inline src/main/java/edu/greenblitz/robotName/commands/Climb/lifter/ExtendLifter.java:25]: Why?

File: src/main/java/edu/greenblitz/robotName/commands/Climb/lifter/ExtendLifter.java [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+            lifter.setPower(0.1);
+        }
+    }
+
+    @Override
+    public boolean isFinished() {
+//        return lifter.isAtPosition(LifterConstants.LIFTER_EXTENDED_POSITION);
```
Comment (yonnbro) [inline src/main/java/edu/greenblitz/robotName/commands/Climb/lifter/ExtendLifter.java:32]: What? Delete comments if not real, but are you sure the command will never finish?

File: src/main/java/edu/greenblitz/robotName/subsystems/Lifter/NeoLifter/NeoLifterConstants.java [hunk 6]
```diff
@@ -18,13 +22,13 @@ public class NeoLifterConstants {
...
     public static final SimpleMotorFeedforward FEED_FORWARD = new SimpleMotorFeedforward(0,0,0);
     
-    public static final SparkLimitSwitch.Type FORWARD_LIMIT_SWITCH_TYPE = SparkLimitSwitch.Type.kNormallyClosed;
+    public static final SparkLimitSwitch.Type FORWARD_LIMIT_SWITCH_TYPE = SparkLimitSwitch.Type.kNormallyOpen;
 
-    public static final SparkLimitSwitch.Type BACKWARD_LIMIT_SWITCH_TYPE = SparkLimitSwitch.Type.kNormallyClosed;
+    public static final SparkLimitSwitch.Type BACKWARD_LIMIT_SWITCH_TYPE = SparkLimitSwitch.Type.kNormallyOpen;
 
-    public static final boolean IS_BACKWARD_LIMIT_SWITCH_ENABLED = true;
+    public static final boolean IS_BACKWARD_LIMIT_SWITCH_ENABLED = false;
```
Comment (yonnbro) [inline src/main/java/edu/greenblitz/robotName/subsystems/Lifter/NeoLifter/NeoLifterConstants.java:29]: Why false???

File: src/main/java/edu/greenblitz/robotName/subsystems/Lifter/LifterSolenoid/RealLifterSolenoid/RealLifterSolenoidConstants.java [hunk 7]
```diff
@@ -0,0 +1,7 @@
+package edu.greenblitz.robotName.subsystems.Lifter.LifterSolenoid.RealLifterSolenoid;
+
+public class RealLifterSolenoidConstants {
+
+	public static final int SOLENOID_ID = 6;
```
Comment (yonnbro) [inline src/main/java/edu/greenblitz/robotName/subsystems/Lifter/LifterSolenoid/RealLifterSolenoid/RealLifterSolenoidConstants.java:5]: No real hold voltage, no power to hold, no power to open, no power to close

File: src/main/java/edu/greenblitz/robotName/subsystems/Lifter/LifterSolenoid/LifterSolenoidConstants.java [hunk 8]
```diff
@@ -0,0 +1,12 @@
+package edu.greenblitz.robotName.subsystems.Lifter.LifterSolenoid;
+
+public class LifterSolenoidConstants {
+
```
Comment (yonnbro) [inline src/main/java/edu/greenblitz/robotName/subsystems/Lifter/LifterSolenoid/LifterSolenoidConstants.java:4]: Wich const class are you using? This or realliftersolonoid?

**Review by yonnbro — CHANGES_REQUESTED** [review]

**Pull request closed without merging**